pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1663,3 +1663,53 @@ def func(x):
     msg = "length must not be 0"
     with pytest.raises(ValueError, match=msg):
         df.groupby("A", observed=False).agg(func)
+
+
+@pytest.mark.parametrize(
+    "aggfunc",
+    [
+        "mean",
+        np.mean,
+        ["sum", "mean"],
+        [np.sum, np.mean],
+        ["sum", np.mean],
+        lambda x: x.mean(),
+        {"A": "mean"},
+        {"A": "mean", "B": "sum"},
+        {"A": np.mean},
+    ],
+    ids=[
+        " string_mean ",
+        " numpy_mean ",
+        " list_of_str_and_str ",
```

Comment:
Why the spaces?

Reply:
Think it was ease for reading for myself when it was showing up on my tests, so it was easier to read

`xxx - yyyy - zzzz`

Will make sure to cleanup before taking off draft marker though.